Tests/LibWeb/Text/input/ServiceWorker/service-worker-register.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<script>
+    asyncTest(done => {
+        let currentURL = document.location.href;
+
+        function finishTest() {
+            println("DONE");
+            internals.spoofCurrentURL(currentURL);
+            done();
+        }
+
+        internals.spoofCurrentURL("https://example.com/service-worker-register.html");
```

Comment:
Could you make a `spoofCurrentURL()`  wrapper function in `include.js` that records what the original URL is and sets the URL back to the original value when the test is done?